compiler/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "grpcio-compiler"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
Actually the changes is incompatible as it changes C interface. So grpcio 0.6.0 can't compile with grpcio-compiler 0.6.1. Maybe we should use a separate branch instead.